### ISSUE TITLE
Critical items all appear as locked 

### ISSUE THREAD

**Issue by PHEN-** [xiprox/WaniKani-for-Android#10]
All my critical items appear as locked from the main dashboard,  and greyed out.  Clicking on them says progress is locked,  but I definitely seen them in reviews.

**Comment by xiprox** [comment]
Well, for an item to be a critical item, it has to be unlocked first and answered wrongly, to begin with. Can you provide me your API key? You can email it to x.ihsan@gmail.com.

**Comment by xiprox** [comment]
Yeah, the API returns `null` for the `user_specific` field for critical items. What I'm doing is: I'm passing the CriticalItem from the list to ItemDetailsActivity. ItemDetailsActivity is set to fetch a complete item if the item passed to it is a CriticalIem or an UnlockItem. Fetching a complete item is done by pulling the whole list of radicals/kanji/vocabulary for the level of the passed item and then finding the passed item in the list. The item then has all the necessary information. I did some debugging yesterday and it seems to be good. Something just doesn't work there though.

Also, I fixed grey items in dashboard, in case you would look into that.
